.travis.yml

```diff
@@ -10,14 +10,13 @@ env:
   -
   - GOFLAGS='-race'
 
-matrix:
-  fast_finish: true
+go_import_path: github.com/google/trillian-examples
```

Comment:
Ooh, that's interesting.  I wonder if this could replace the [hacks](https://github.com/google/certificate-transparency-go/blob/master/.travis.yml#L20-L23) in the other repos?

Reply:
Maybe. The docs for it don't give too much away.

Reply:
Looks like it works on the main repo as well.